aws/docker-pull.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Intended for use on the initial build of a box. Ansible will update/restart
+# if changed subseqsently.
+
+IMAGES=(
+  jrgm/123done
+  mozilla/fxa-auth-db-mysql
```

Comment:
Just to clarify, this script is only used to bootstrap. Updating of docker images (and container restart if changed) is handled by ansible (run in a periodic cronjob).